Mathlib/RingTheory/Adjoin/Tower.lean

```diff
@@ -164,7 +164,8 @@ theorem fg_of_fg_of_fg [IsNoetherianRing A] (hAC : (⊤ : Subalgebra A C).FG)
   Algebra.fg_trans' (B₀.fg_top.2 hAB₀) <|
     Subalgebra.fg_of_submodule_fg <|
       have : IsNoetherianRing B₀ := isNoetherianRing_of_fg hAB₀
-      have : IsNoetherian B₀ C := isNoetherian_of_fg_of_noetherian' hB₀C
+      have : Module.Finite B₀ C := ⟨hB₀C⟩
+      have : IsNoetherian B₀ C := isNoetherian_of_isNoetherianRing_of_finite B₀ C
       fg_of_injective (IsScalarTower.toAlgHom B₀ B C).toLinearMap hBCi
 #align fg_of_fg_of_fg fg_of_fg_of_fg
```

Comment:
This lemma should probably replace `(⊤ : Submodule B C).FG` for `Module.Finite B C`, though that probably makes sense for a separate PR.

Reply:
Yes, I was going to do it until I realised this entire file needs such a change made (and appropriate renames etc). Worth doing but I'll defer for now.